rc/tools/make.kak

```diff
@@ -1,7 +1,9 @@
 declare-option -docstring "shell command run to build the project" \
     str makecmd make
-declare-option -docstring "pattern that describes lines containing information about errors in the output of the `makecmd` command" \
+declare-option -docstring "pattern that find lines containing information about errors in the output of the `makecmd` command" \
     str make_error_pattern " (?:fatal )?error:"
+declare-option -docstring "pattern that describes lines containing information about errors in the output of the `makecmd` command" \
+    str make_error_line_pattern ""
```

Comment:
I guess we should use `regex` as type instead of `str`. Then we get early regex validation

Also we should add to the docstring something like

```
Capture groups must be:
    1: filename
    2: line number
    3: optional column
```

Reply:
done, thanks